stubs/numpy.py

```diff
@@ -0,0 +1,178 @@
+from typing import (
+    Any,
+    Tuple,
+    Type,
+    TypeVar,
+    Union,
+)
+
+from mypy_extensions import (
+    TypedDict,
+)
+
+T = TypeVar('T')
+
+
+class TErrorOptions(TypedDict):
```

Comment:
Is there a reason to use a typed dict instead of a named tuple?  My initial reaction is that there's no real advantage since the performance characteristics of named tuples are inherently better and they seem to provide all the needed functionality.

Reply:
This isn't our API but rather numpy's.  I used `TypedDict` so that we could have stronger safety on the actual return type.

https://docs.scipy.org/doc/numpy/reference/generated/numpy.seterr.html